src/parser.rs

```diff
@@ -746,6 +746,7 @@ fn parse_pointer_definition(
         (
             DataTypeDeclaration::DataTypeDefinition {
                 data_type: DataType::PointerType { name, referenced_type: Box::new(decl), auto_deref },
+                // FIXME: this currently includes the initializer in the sourcelocation, resulting in 'REF_TO A := B' when creating a slice
```

Comment:
Should we track this on GitHub instead in the code itself? I feel like these FIXMEs are easily forgotten